pkgs/development/python-modules/trl/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  setuptools-scm,
+  accelerate,
+  datasets,
+  rich,
+  transformers,
+}:
+
+buildPythonPackage rec {
+  pname = "trl";
+  version = "0.15.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "huggingface";
+    repo = "trl";
+    tag = "v${version}";
+    hash = "sha256-HsSmFXFqDOWVLa6VXdPZVS9C3bjYcsliR0TwNpPiQx4=";
+  };
+
+  build-system = [ setuptools ];
+  nativeBuildInputs = [ setuptools-scm ];
+
+  dependencies = [
+    accelerate
+    datasets
+    rich
+    transformers
+  ];
+
+  doCheck = true;
```

Comment:
can you enable the tests with
```
nativeCheckInputs = [unittestCheckHook];
```

Reply:
Enabling the tests requires to download models from https://huggingface.co.